generator/internal/dart/annotate.go

```diff
@@ -229,7 +229,7 @@ func (annotate *annotateModel) annotateModel(options map[string]string) (*modelA
 	ann := &modelAnnotations{
 		PackageName:    packageName(model, packageNameOverride),
 		PackageVersion: packageVersion,
-		MainFileName:   strcase.ToSnake(model.Name),
+		MainFileName:   model.Name,
```

Comment:
Without this change the `iam_v1.dart` would come out as `iam_v_1.dart`; I could perhaps keep the ToSnake() here but introduce a way to override the main file name (as we do for package names - `package-name-override`).